src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ endif()
 set(TARGET_NAME "openvino_intel_cpu_plugin")
 
 if((CMAKE_COMPILER_IS_GNUCXX OR OV_COMPILER_IS_CLANG) AND CMAKE_CXX_STANDARD GREATER_EQUAL 20)
-    set(CMAKE_CXX_FLAGS "-Wno-error=deprecated ${CMAKE_CXX_FLAGS}")
+    set(CMAKE_CXX_FLAGS "-Wno-error=deprecated -Wno-alloc-size-larger-than ${CMAKE_CXX_FLAGS}")
```

Comment:
I suppose it's available only with newer compiler version? should be suppress it conditionally? maybe we can suppress inside the code directly to localize suppression?

Reply:
Fixed in code instead